### PR TITLE
Added changes for samples to run in MetaOS hubs.

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/msgext-search-quickstart/js/README.md [hunk 1]
```diff
@@ -11,96 +11,102 @@ languages:
 extensions:
  contentType: samples
  createdDate: "07/07/2021 01:38:27 PM"
-urlFragment: officedev-microsoft-teams-samples-msgext-search-quickstart-js
+urlFragment: officedev-microsoft-teams-samples-msgext-action-quickstart-js
```
Comment (ChetanSharma-msft) [inline samples/msgext-search-quickstart/js/README.md:14]: Please verify if it's properly updated in sample.json.

Comment (ChetanSharma-msft) [inline samples/msgext-search-quickstart/js/README.md:14]: This sample is related to msgext-search as per sample name.?

File: samples/msgext-search-quickstart/js/README.md [hunk 2]
```diff
@@ -11,96 +11,102 @@ languages:
...
 ## Included Features
 * Bots
 * Message Extensions
-* Search Commands
+* Action Commands
 
 ## Interaction with app
 
-![Sample Module](Images/msgextsearchquickstart.gif)
+![message ext module](Images/msgextactionquickstartModule.gif)
```
Comment (ChetanSharma-msft) [inline samples/msgext-search-quickstart/js/README.md:28]: Please give proper name.  "message **ext** module"

File: samples/msgext-search-quickstart/js/README.md [hunk 3]
```diff
@@ -11,96 +11,102 @@ languages:
...
+ - Clone the repository
 
     ```bash
     git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
     ```
-
-  - In a terminal, navigate to `samples/msgext-search-quickstart/js`
-  
-  - Update the `.env` configuration for the bot to use the `MicrosoftAppId` and `MicrosoftAppPassword`, `BaseUrl` with application base url. For e.g., your ngrok url. (Note the MicrosoftAppId is the AppId created in step 1 (Setup for Bot), the MicrosoftAppPassword is referred to as the "client secret" in step 1 (Setup for Bot) and you can always create a new client secret anytime.)
+ -In a terminal, navigate to `samples/msgext-action-quickstart/js`
```
Comment (ChetanSharma-msft) [inline samples/msgext-search-quickstart/js/README.md:69]: search or action?

File: samples/msgext-search-quickstart/js/appPackage/manifest.json [hunk 4]
```diff
@@ -15,12 +15,12 @@
...
   "name": {
-    "short": "Messaging Extension Search",
+    "short": "Messaging Extension Action",
     "full": ""
   },
   "description": {
-    "short": "Short description for Messaging Extension Search.",
-    "full": "Full description of Messaging Extension Search."
+    "short": "Short description for Messaging Extension Action.",
+    "full": "Full description of Messaging Extension Action."
```
Comment (ChetanSharma-msft) [inline samples/msgext-search-quickstart/js/appPackage/manifest.json:23]: App folder name is msgext-search-...

File: samples/msgext-search-quickstart/js/appPackage/manifest.json [hunk 5]
```diff
@@ -56,6 +86,6 @@
     "messageTeamMembers"
   ],
   "validDomains": [
-    "<<valid-domain>>"
+    "localhost:3000"
```
Comment (ChetanSharma-msft) [inline samples/msgext-search-quickstart/js/appPackage/manifest.json:89]: Are we not going to add ngrok URL here?

You have removed `"<<valid-domain>>"`

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added review comments.

File: samples/msgext-search-quickstart/js/README.md (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/msgext-search-quickstart/js/README.md]: changed

Comment (Prithvi-MSFT) [reply samples/msgext-search-quickstart/js/README.md]: yes

File: samples/msgext-search-quickstart/js/README.md (hunk 2 above)
Comment (Prithvi-MSFT) [reply samples/msgext-search-quickstart/js/README.md]: changed

File: samples/msgext-search-quickstart/js/README.md (hunk 3 above)
Comment (Prithvi-MSFT) [reply samples/msgext-search-quickstart/js/README.md]: changed

File: samples/msgext-search-quickstart/js/appPackage/manifest.json (hunk 5 above)
Comment (Prithvi-MSFT) [reply samples/msgext-search-quickstart/js/appPackage/manifest.json]: changed

File: samples/msgext-search-quickstart/js/appPackage/manifest.json (hunk 4 above)
Comment (Prithvi-MSFT) [reply samples/msgext-search-quickstart/js/appPackage/manifest.json]: changed

**Comment by ChetanSharma-msft** [comment]
@Prithvi-MSFT - Please run build pipeline for this.

**Review by Harikrishnan-MSFT — APPROVED** [review]

**Comment by Prithvi-MSFT** [comment]
Build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5179746157

**Review by Hisakaq8 — COMMENTED** [review]
[https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/823#pullrequestreview-1461841151](url)

**Pull request merged**